timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -144,55 +145,55 @@ public void nonLeadersReturn503() {
 
     @Test
     public void leaderRespondsToRequests() {
-        CLUSTER.currentLeader().getFreshTimestamp();
+        cluster.currentLeader().getFreshTimestamp();
 
-        LockToken token = CLUSTER.currentLeader().lock(LockRequest.of(LOCKS, DEFAULT_LOCK_TIMEOUT_MS)).getToken();
-        CLUSTER.unlock(token);
+        LockToken token = cluster.currentLeader().lock(LockRequest.of(LOCKS, DEFAULT_LOCK_TIMEOUT_MS)).getToken();
+        cluster.unlock(token);
     }
 
     @Test
     public void newLeaderTakesOverIfCurrentLeaderDies() {
-        CLUSTER.currentLeader().kill();
+        cluster.currentLeader().kill();
 
-        CLUSTER.getFreshTimestamp();
+        cluster.getFreshTimestamp();
     }
 
     @Test
     public void leaderLosesLeadershipIfQuorumIsNotAlive() {
-        TestableTimelockServer leader = CLUSTER.currentLeader();
-        CLUSTER.nonLeaders().forEach(TestableTimelockServer::kill);
+        TestableTimelockServer leader = cluster.currentLeader();
+        cluster.nonLeaders().forEach(TestableTimelockServer::kill);
 
         assertThatThrownBy(leader::getFreshTimestamp)
                 .satisfies(ExceptionMatchers::isRetryableExceptionWhereLeaderCannotBeFound);
     }
 
     @Test
     public void someoneBecomesLeaderAgainAfterQuorumIsRestored() {
-        CLUSTER.nonLeaders().forEach(TestableTimelockServer::kill);
-        CLUSTER.nonLeaders().forEach(TestableTimelockServer::start);
+        cluster.nonLeaders().forEach(TestableTimelockServer::kill);
+        cluster.nonLeaders().forEach(TestableTimelockServer::start);
 
-        CLUSTER.getFreshTimestamp();
+        cluster.getFreshTimestamp();
     }
 
     @Test
     public void canPerformRollingRestart() {
         bringAllNodesOnline();
-        for (TestableTimelockServer server : CLUSTER.servers()) {
+        for (TestableTimelockServer server : cluster.servers()) {
             server.kill();
-            waitForClusterToStabilize();
-            CLUSTER.getFreshTimestamp();
+//            waitForClusterToStabilize();
```

Comment:
Is this intentional? I'm not sure how long start/kill take, but this seems like it could be dangerous if we kill the leader and then try and get a fresh timestamp, especially if one of the other nodes is just starting up. I haven't looked at how `start` works, so it's possible it's not needed

(If definitely not needed, then the comment should probably be removed - and possibly one added on why this is safe)

Reply:
technically not needed, tests passed just fine, but I'd rather not reason about it, so I just inlined what the previous thing did into this method. The only thing I removed was the ability to do it as a `@ClassRule` which can be a bit awkward but I think it's fine. I'm not sure what the value is (in the current impl) of waiting for "leadership" of multiple clients at the beginning of the test (suite), and doesn't seem to affect results 🤷‍♂